lib/recipe_box/cuisine.ex

```diff
@@ -42,5 +40,15 @@ defmodule RecipeBox.Cuisine do
     |> Cuisine.changeset(attrs)
     |> Repo.insert()
   end
+
+  def list_cuisines() do
+    Repo.all(Cuisine)
+  end
+
+  def alphabetize_cuisines(cuisines) do
```

Comment:
nit: this is really returning alphabetized cuisine _names_—the rest of the data is dropped.